src/formatter/helpers/gherkin_document_parser_spec.ts

```diff
@@ -315,64 +316,64 @@ async function parseGherkinDocument(data: string): Promise<IGherkinDocument> {
 }
 
 async function withBackgroundAndScenario(): Promise<IGherkinDocument> {
-  return await parseGherkinDocument(`\
-Feature: a feature
-  Background:
-    Given a setup step
-
-  Scenario:
-    When a regular step
-`)
+  return await parseGherkinDocument(unindent`
```

Comment:
I've never seem this syntax. Is this a function call? If so, I think i'd prefer we add parens so it is
unindent(`...`)

Reply:
This is a tagged template, a JavaScript feature that is part of the template literals spec

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates

Reply:
Is there any reason we are using tagged template vs generic function? Seems like it would be more approachable / easier to understand with just a function

Reply:
For me it feels that this is a perfect use-case for tag templates. There is no hack, nothing has been worked-around for this.

Tagged templates arrived alongside template strings with es6 (es-2015), are perfectly documented and have the same support, and almost any article on the Internet which introduce template literals also introduces tag templates.

I cannot see any reason not to use tag template here actually 😕

Reply:
In my opinion, the fact that you have to reconstruct the interpolated string by adding the expression values and strings is unnecessary complexity being exposed to us. If this was a function that just took in the interpolated string, that logic could be removed. It would be strictly less code.

Tag templates to me are more about controlling the interpolation in some way which is not something we are using here.

Reply:
That is a very good point 😅 

I change it for a function 👍